### PR TITLE
Updating roadmap

### PR DESCRIPTION
**Pull request by rladuca**
Adding components that have not yet been published in their priority order.

### CODE REVIEW

**Comment by psmulovics** [comment]
so would milcore be added to the list?

**Comment by dotMorten** [comment]
I'm guessing that's the "PresentationNative_cor" one?

**Comment by rladuca** [comment]
milcore is WpfGfx_cor3, so yes it will be published eventually.

File: roadmap.md [hunk 1]
```diff
@@ -22,9 +22,46 @@ As we complete those goals, we'll update our roadmap to include additional featu
 
 ## Porting Status
 
-The port from WPF for .NET Framework is still in progress. Currently this repository contains these components:
+The port from WPF for .NET Framework is still in progress.  All components applicable to WPF for .NET Core will eventually be published to this repository.
```
Comment (tomashubelbauermicrosoft) [inline roadmap.md]: Two spaces :-)

**Pull request merged**